### ISSUE TITLE
Camera orientation metadata filter

### ISSUE THREAD

**Issue by Beep6581** [Beep6581/RawTherapee#2315]
Originally reported on Google Code with ID 2331

```
Feature request for a metadata filter based on camera orientation.

exiv2:
Exif.Image.Orientation

exiftool:
-EXIF:Orientation=Horizontal
-EXIF:Orientation=Rotate 270 CW
-EXIF:Orientation=Unknown (9)
```

Reported by `entertheyoni` on 2014-04-06 09:14:35

**Comment by Beep6581** [comment]
Related to #210

**Comment by heckflosse** [comment]
Closed with ddfce7e
